lib/keycloak.rb

```diff
@@ -255,8 +255,7 @@ def user_signed_in?(access_token)
 
     def user_token(uname_email, password)
       user = user_by_email(uname_email)
-      valid = user && user.dig(:credentials, :password) == password
-      raise FailedRequestError.new 'Failed to set user password' unless valid
+      valid = user && user.dig(:credentials, :value) == password
```

Comment:
Why bother checking if we do nothing with the result? I do not understand that change.

Reply:
ok - the check is back in 1c2025a7e4da887845d511ae8019d29e2316a3b9 ;)